src/utils/numberUtils.js

```diff
@@ -4,15 +4,20 @@
  * @param {number} value
  * @param {number} decimals How many decimals after the separator must be present after rounding
  *   (default to 0)
+ * @param {boolean} enforcedigit if set to true, we want to have that many figures after the period.
+ *   Otherwise, we don't care.
  * @returns {number} Value rounded
  */
-export function round(value, decimals = 0) {
+export function round(value, decimals = 0, enforcedigit=false) {
     if (!isNumber(value)) {
         return undefined
     }
     if (decimals === 0) {
         return Math.round(value)
     }
+    if (enforcedigit) {
+    return value.toFixed(decimals)
+    }
```

Comment:
linting 😉 (missing tab here)
you should be able to auto-lint with Prettier, if you use VSCode or JetBrains products, you can add Prettier as a save action command (or there's typically a shortcut to launch it)